src/pattern-library/components/patterns/SpinnerPatterns.js

```diff
@@ -21,23 +26,23 @@ export default function SpinnerPatterns() {
             At its default size, the spinner is <code>2em</code> square.
           </p>
           <Library.Demo withSource>
-            <SvgIcon name="hyp-spinner" className="hyp-spinner" />
+            <Icon name="hyp-spinner" containerClasses="hyp-spinner" />
```

Comment:
The notes on the "Hypothesis UI playground" homepage say that the Patterns are built "using HTML and CSS" whereas components "are Preact components that are built using underlying patterns. I note that we are using `Icon` here, which I guess is OK assuming in a non-JS context we'd have some other way to render the same SVG icon (like the Jinja2 helper that h has). I'm also ignoring the usage of components which are just helpful for the demo page and not required to actually use the pattern.

Reply:
You're right! This is non-normalized. There's a bit of work here because the offending bit is actually that we don't have a "pattern" established for `icon`, just a component. I'm going to track this for follow-up work.